app/assess/data.py

```diff
@@ -115,6 +118,49 @@ def get_available_tags_for_fund_round(fund_id, round_id) -> List[Tag]:
         return []
 
 
+def get_tag_for_fund_round(fund_id, round_id, tag_id) -> List[Tag]:
+    endpoint = Config.ASSESSMENT_TAG_ENDPOINT.format(
+        fund_id=fund_id, round_id=round_id, tag_id=tag_id
+    )
+    response = get_data(endpoint)
+    if response is not None:
+        current_app.logger.info(f"tag returned: {len(response)}")
+        result = Tag.from_dict(response)
```

Comment:
This isn't returning `List[Tag]` - should this be nested in a list?

Reply:
Or the type annotation and `return []` need updated?

Reply:
good spot